lib/my_john_deere_api/accessor.rb

```diff
@@ -0,0 +1,45 @@
+module MyJohnDeereApi
+  class Accessor
+    attr_reader :access_token
+
+    REQUEST_METHODS = [:get, :post, :put, :delete]
+
+    RETRY_DELAY_EXPONENT = 2
+    MAX_RETRIES = 12
```

Comment:
Is this configurable? When we make API requests in a job, we probably don't want to wait the full 34 minutes, especially when Sidekiq already has an exponential retry mechanism.

Reply:
I agree, I was thinking that all hardcoded parts should be defaults which can be overridden with an options hash. I'm going to do that.